### ISSUE TITLE
Discontinuous Exodus in initial_from_file_var

### ISSUE THREAD

**Issue by loganharbour** [idaholab/moose#13200]
## Bug Description
It appears that the loading of discontinuous variables via `initial_from_file_var`, whether or not they are saved with `discontinuous = true`, is not well defined. I've spent a fair share of time looking through old discussions, primarily those that lead to #11210. 

## Steps to Reproduce
With the mesh [l2.e.zip](https://github.com/idaholab/moose/files/3060620/l2.e.zip) (2x2 grid with a variable `l2_lagrange` of type `FIRST L2_LAGRANGE`) and the following input:
```
[Mesh]
  type = FileMesh
  file = l2.e
[]

[Variables]
  [./l2_lagrange]
    family = L2_LAGRANGE
    order = FIRST
    initial_from_file_var = l2_lagrange
  [../]
[]

[Kernels]
  [./diff]
    type = NullKernel
    variable = l2_lagrange
  []
[]

[Executioner]
  type = Steady
[]
```
the result is:
```
Available variables:
Unable to locate variable named: l2_lagrange
Stack frames: 11
0: 0   libmesh_opt.0.dylib                 0x000000011046185e libMesh::print_trace(std::__1::basic_ostream<char, std::__1::char_traits<char> >&) + 1198
1: 1   libmesh_opt.0.dylib                 0x000000011045d74e libMesh::MacroFunctions::report_error(char const*, int, char const*, char const*) + 270
2: 2   libmesh_opt.0.dylib                 0x00000001106c6b59 libMesh::ExodusII_IO_Helper::read_elemental_var_values(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, int, std::__1::map<unsigned int, double, std::__1::less<unsigned int>, std::__1::allocator<std::__1::pair<unsigned int const, double> > >&) + 1113
3: 3   libmesh_opt.0.dylib                 0x00000001106b5466 libMesh::ExodusII_IO::copy_elemental_solution(libMesh::System&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, unsigned int) + 118
```

Venturing into `SystemBase` [here](https://github.com/idaholab/moose/blob/next/framework/src/systems/SystemBase.C#L941) we see that `copy_elemental_solution` is called, as expected.

## Impact
Unintended behavior for loading a standard variable type. Not sure how significant of an issue this is, but it definitely complicates importing such a type.

I wouldn't mind taking a crack at this, but I'm really not sure the best way to go about it. Do I keep MOOSE as it is and modify [`copy_elemental_solution`](https://github.com/libMesh/libmesh/blob/master/src/mesh/exodusII_io.C#L512-L552)? Do I consider it a nodal value and modify [`copy_nodal_solution`](https://github.com/libMesh/libmesh/blob/master/src/mesh/exodusII_io.C#L478-L508)? I would think to check in which ever of the functions above is deemed appropriate if the system variable is of a discontinuous type and then do something different as needed.

**Comment by YaqiWang** [comment]
Exodus just does not save all the solution information directly. It saves high order elemental variables either in constant monomial or break the mesh and save them on each small piece with nodal variables if I understand correctly.

**Comment by loganharbour** [comment]
Yes, I definitely agree and understand: see [here](https://github.com/libMesh/libmesh/blob/master/src/mesh/exodusII_io.C#L1249-L1303). I guess this question is: (1) is there enough here to reconstruct it and (2) is it even worth it.

**Comment by permcody** [comment]
I would suggest fixing this at the reader level. This bug would exist in libMesh too so it should be fixed there.

**Comment by GiudGiud** [comment]
This came up in #17574.
Once this issue is resolved I think we should update the documentation to say this is not supported.
